axelar-chains-config/info/mainnet.json

```diff
@@ -3468,6 +3468,7 @@
     "grpc": "grpc.axelar.lava.build:443",
     "tokenSymbol": "AXL",
     "axelarscanApi": "https://api.axelarscan.io",
+    "gmpAxelarscanApi": "https://api.gmp.axelarscan.io",
```

Comment:
Is this actually needed? We already have the API above. As per docs, you can use the path `{API}/gmp` for the request

Reply:
good catch, i updated endpoint to be `/gmp/estimateITSFee` and that fixed it so will remove this.